tcms/testruns/tests/test_admin.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+
+from http import HTTPStatus
+
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from tcms.tests import LoggedInTestCase
+from tcms.tests.factories import UserFactory
+
+
+class TestSuperuserActions(LoggedInTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.superuser = UserFactory()
+        cls.superuser.is_staff = True
+        cls.superuser.is_superuser = True
+        cls.superuser.set_password('password')
+        cls.superuser.save()
+
+        cls.regular_user = UserFactory()
+        cls.regular_user.is_staff = True
+        cls.regular_user.set_password('password')
+        cls.regular_user.save()
+
+        cls.url = reverse('admin:auth_user_delete', args=[cls.regular_user.pk])
```

Comment:
Delete this entire method b/c we don't need it for now and b/c you keep adding thing which are not in scope and which don't work properly.